commands/command.go

```diff
@@ -12,8 +12,8 @@ import (
 type Command interface {
```

Comment:
The name `Command` seems to be a misnomer and a little misleading. Something like `Operation` would be better. Don't bother now - something to think about at later stage.